### ISSUE TITLE
tests.common.test_format_utils.TestFormatTimestamp fails on Ubuntu Trusty

### ISSUE THREAD

**Issue by mjeanson** [lttng/lttng-analyses#52]
On an Ubuntu Trusty 14.04, the tests.common.test_format_utils.TestFormatTimestamp fails with a delta of one second. The tzdata version on this system is 2016d-0ubuntu0.14.04 while it works on Xenial with tzdata 2016c-1.

```
======================================================================
FAIL: test_negative (tests.common.test_format_utils.TestFormatTimestamp)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/root/lttng-analyses/tests/common/test_format_utils.py", line 150, in test_negative
    self.assertEqual(result, '1948-05-08 23:02:51.876543211')
AssertionError: '1948-05-08 23:02:52.876543211' != '1948-05-08 23:02:51.876543211'
- 1948-05-08 23:02:52.876543211
?                   ^
+ 1948-05-08 23:02:51.876543211
?                   ^

```

This seems to indicate a leap second may have been introduced in this new tzdata, not sure what we can do about this.

**Comment by mjeanson** [comment]
After looking at the changelog for the tzdata package, it may have more to do with the version of python / dateutils.

**Comment by abusque** [comment]
Fixed in 3d449a5
